package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@uniswap/smart-order-router",
-  "version": "3.20.2",
+  "version": "3.23.0",
```

Comment:
let's bump SOR version in a separate, dedicated PR, like https://github.com/Uniswap/smart-order-router/pull/491.

And I think we need to bump to v3.25.0 in that dedicated PR.